contract/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Contracts Management - Recurring',
-    'version': '10.0.1.1.0',
+    'version': '10.0.1.1.1',
```

Comment:
Module version should raise at least to `10.0.1.2.0`, but this can be considered a huge change and it should be `10.0.2.0.0`